src/gui/DatabaseOpenWidget.cpp

```diff
@@ -27,11 +27,19 @@
 #include "format/KeePass2Reader.h"
 #include "keys/FileKey.h"
 #include "keys/PasswordKey.h"
+#include "crypto/Random.h"
+#include "keys/YkChallengeResponseKey.h"
+
+#include "config-keepassx.h"
+
+#include <QtConcurrentRun>
+#include <QSharedPointer>
+
 
 DatabaseOpenWidget::DatabaseOpenWidget(QWidget* parent)
-    : DialogyWidget(parent)
-    , m_ui(new Ui::DatabaseOpenWidget())
-    , m_db(nullptr)
+    : DialogyWidget(parent),
+      m_ui(new Ui::DatabaseOpenWidget()),
+      m_db(nullptr)
 {
```

Comment:
it's ugly but functional, makes it easy to add/remove items at the end of the list